packages/extension-api/src/extension-api.d.ts

```diff
@@ -401,6 +401,10 @@ declare module '@podman-desktop/api' {
      * the vmTypeDisplayName property cannot be set if vmType is undefined
      */
     vmTypeDisplayName?: string;
+    /**
+     * the remote may be different from the client version
+     */
+    version?(): string | undefined;
```

Comment:
Question: I don't get why it is a method ?

why not like vmType ? version?: string

Reply:
So the problem is the same as the `status`

https://github.com/podman-desktop/podman-desktop/blob/d091c29966f7f94ea3681145fdacdb9ebe64a66f/packages/extension-api/src/extension-api.d.ts#L398C5-L398C12

The version cannot be fetch if the machine is not running, so I used the same logic as the status

Reply:
Can't we have it as a property that will get lazy initialized when machine is started so that we can get the info even if the machine is stopped ?